client-ts/signalr-protocol-msgpack/src/third-party-notices.txt

```diff
@@ -8,6 +8,85 @@ bring it to our attention. Post an issue or email us:
 
 The attached notices are provided for information only.
```

Comment:
Paging @Eilon

Reply:
This looks good.

Reply:
And just to clarify, the `src` directory **is** included in the final package (it's useful for source maps, debugging, etc.). If we decide to remove it, we'll need to ensure this file remains somewhere in the final package.

Reply:
Yup.